php/class-coauthors-guest-authors.php

```diff
@@ -1503,4 +1506,95 @@ public function filter_author_feed_link( $feed_link, $feed ) {
 
 		return $link;
 	}
+
+	/**
+	 * Filter Personal Data Exporters to add Guest Author exporter
+	 *
+	 * @since 3.3.1
+	 */
+	public function filter_personal_data_exporter( $exporters ) {
+		$exporters['cap-guest-author'] = array(
+			'exporter_friendly_name' => __( 'Guest Author', 'co-authors-plus' ),
+			'callback'               => array( $this, 'personal_data_exporter' ),
+		);
+
+		return $exporters;
+	}
+
+	/**
+	 * Finds and exports personal data associated with an email address for guest authors
+	 *
+	 * @since 3.3.1
+	 *
+	 * @param string $email_address  The guest author email address.
+ 	 * @return array An array of personal data.
+	 */
+	public function personal_data_exporter( $email_address ) {
+		$email_address = trim( $email_address );
+
+		$data_to_export = array();
+
+		$author = $this->get_guest_author_by( 'user_email', $email_address );
+
+		if ( ! $author ) {
+			return array(
+				'data' => array(),
+				'done' => true,
+			);
+		}
+
+		$author_data = array(
+			'ID'              => __( 'ID', 'co-authors-plus' ),
+			'user_login'      => __( 'Login Name', 'co-authors-plus' ),
+			'display_name'    => __( 'Display Name', 'co-authors-plus' ),
+			'user_email'      => __( 'Email', 'co-authors-plus' ),
+			'first_name'      => __( 'First Name', 'co-authors-plus' ),
+			'last_name'       => __( 'Last Name', 'co-authors-plus' ),
+			'website'         => __( 'Website', 'co-authors-plus' ),
+			'aim'             => __( 'AIM', 'co-authors-plus' ),
+			'yahooim'         => __( 'Yahoo IM', 'co-authors-plus' ),
+			'jabber'          => __( 'Jabber / Google Talk', 'co-authors-plus' ),
+			'description'     => __( 'Biographical Info', 'co-authors-plus' ),
+		);
+
+		$author_data_to_export = array();
+
+		foreach ( $author_data as $key => $name ) {
+			if ( empty( $author->$key ) ) {
+                continue;
+            }
```

Comment:
Looks like the indentation is off here too.

Reply:
Sublime is playing with me hahaha

I guess we should consider to add a `.editoconfig`